### DISCUSSION TITLE
feature request: Interactions

### DISCUSSION THREAD

**Discussion by Ombucha** [stoatchat/discussions#483] [Feature Suggestions]
### What do you want to see?

- Message Components (Selects, Modals & Buttons)
- Application Commands (Slash, Message & User)

### PWA

- [ ] Yes, this feature request is specific to the PWA.

**Comment by aurkaxi** [comment]
+1 for both
Slash commands makes it easier to take arguments
No need to talk about message components, we all love them.

**Comment by vico93** [comment]
Looking forward this! Specially the modals components. Would love to see a counterpart of [Forms bot](https://top.gg/bot/941460895912034335) be feasible on Revolt.

**Comment by Garlayntoji** [comment]
Bumping the thread, as a bot developer that uses slash commands it'd be easier to migrate my bot.
